psl/diagnostics/src/warning.rs

```diff
@@ -25,6 +25,15 @@ impl DatamodelWarning {
         Self::new(message, span)
     }
 
+    pub fn new_missing_index_on_emulated_relation(span: Span) -> DatamodelWarning {
+        let message = indoc! {
+            r#"With `relationMode = \"prisma\"`, no foreign keys are used, so `@relation`'s fields will not benefit from the index usually created by the relational database under the hood.
+            This can lead to poor performance when querying these fields. We recommend adding an index manually.
+            Learn more at https://pris.ly/d/relation-mode-prisma#indexes"#
```

Comment:
This is the wrong pris.ly link. Fix:
```suggestion
            Learn more at https://pris.ly/d/relation-mode#indexes"#
```